### ISSUE TITLE
add term anteriorly displaced ear

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10332]
**Preferred term label:**
Anteriorly displaced ears

**Synonyms**
-

**Definition (free text, please give PubMed ID)**
PMID37075751

**Parent term (use hpo.jax.org/app)**
Abnormal location of ears HP:0000357

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM 164210
For example: PMID37075751

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl 
PMID:37075751 does not appear to feature anteriorly displaced ears? Is there a Figure?

**Comment by lisannesw** [comment]
Anteriorly displaced ears is mentioned in the case reports in the supplemental information of PMID:37075751. Sadly, there is no figure.

**Comment by pnrobinson** [comment]
@lisannesw  Given that this seems to be the only mention ever of "anteriorly displaced ears" there is no way of knowing if it isn't a typo or other error. I would suggest we not make a term at this time. If we can find proper documentation of the finding including either a detailed text description or a photograph, we should then make a term!
